modules/ROOT/pages/configure-adv-tls-context.adoc

```diff
@@ -0,0 +1,93 @@
+. Select *Users*.
+. Locate the row containing your username and then select the blue link in the *Username* column.
+. In the *Runtime Manager* tab, enable the *Manage Runtime Fabrics* permission for your Runtime Fabric environment:
+
+.. Select *Environment* and then select your environment from the list.
+.. In the *Permissions* section, select *Select access*.
+.. Select *Select all*.
++
+[NOTE]
+It can take up to five minutes for any permissions you add to propagate.
+
+== Add a TLS Context to a Secrets Group
+
+Before you can import the TLS context from the secrets manager, you must add a TLS context.
+
+Adding a TLS context to your secrets group involves supplying a name, target, security information, version, keystore, and, optionally, truststore. 
+
+. In the *Secret Groups* list view, select the secrets group to which you want to add a TLS context. 
+. Select *TLS Context* in the menu on the left, and click *Add TLS Context*. 
+. In the Create TLS Context screen, add the required information:
++
+* *Name* +
+Enter a name for your TLS context. 
+* *Target* +
+Select *Anypoint Security* to validate the SSL handshake for Runtime Fabric. 
+* *TLS Version* +
+By default, TLS 1.2 and TLS 1.3 are selected. +
+For information about the advantages of TLS 1.3, see <<tls_support_rtf,TLS Support on Runtime Fabric>>.
+* *Keystore* +
+From the drop-down list, select the keystore to store the TLS context. +
+For information about adding a keystore to the secrets manager, see xref:anypoint-security::asm-secret-group-creation-task#add_keystore.adoc[Create a Secrets Group]. +
+You can select keystores that are either PEM or non-PEM types. 
+* Optionally select a truststore to which to add the TLS context if you are using a truststore to store certificates trusted by the client. +
+A truststore is required if you are going to enable client authentication.
+* *Expiration Date* +
+Optionally select the expiration date for the certificate.
+. Optionally xref:anypoint-security::enable-client-authentication.adoc[enable client authentication]. 
+. Optionally xref:anypoint-security::cipher-suites.adoc[select ciphers]. 
+. Click *Save*.
+
+[[import_tls_rtf]]
+== Import a TLS Context to Runtime Fabric
+
+. In *Runtime Manager > Runtime Fabrics > Inbound Traffic > Add certificates*, select *Import from Secrets Manager*.
++
+This option imports a TLS context from the secrets manager, and supports advanced configuration such client authentication (mutual authentication), selecting ciphers, and selecting TLS versions.
+. Select the environment the secrets group that contains the TLS context is associated with.
+. Select the secrets group where the TLS context is configured.
+. Select the TLS context to use for Runtime Fabric. +
+You can click *View Details* to see details of the TLS context you are importing. 
```

Comment:
Consider making this a final step:
```suggestion
. Optionally click *View Details* to see details of the TLS context you are importing. 
```
Is this view static or does it show what is happening during the import?
Are we missing a step to actually start the import?

Reply:
BREAK here to submit work to get started.